### PR TITLE
fix(api/web/resource): fix resource import

### PR DESCRIPTION
**Pull request by Han-Ya-Jun**
### Description
- fix resource import

Fixes # (issue)

### Checklist

- [x] 填写 PR 描述及相关 issue (write PR description and related issue)
- [x] 代码风格检查通过 (code style check passed)
- [x] PR 中包含单元测试 (include unit test)
- [x] 单元测试通过 (unit test passed)
- [x] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: src/dashboard/apigateway/apigateway/apis/web/resource/views.py [hunk 1]
```diff
@@ -426,44 +427,79 @@ def post(self, request, *args, **kwargs):
...
             gateway=request.gateway,
-            resources=openapi_manager.get_resource_list(),
-            selected_resources=slz.validated_data.get("selected_resources"),
+            resources=resource_list,
             need_delete_unspecified_resources=False,
             username=request.user.username,
         )
         importer.import_resources()
+        # 如果生成文档还要再生成文档
+        if slz.validated_data.get("doc_language"):
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apis/web/resource/views.py:458]: doc_language 非空即代表要生成文档？ 加一个单独的开关字段更好一些

**Review by wklken — APPROVED** [review]

File: src/dashboard/apigateway/apigateway/apis/web/resource/views.py (hunk 1 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/apis/web/resource/views.py:458]: done 这个暂时和之前的交互保持一致的

File: src/dashboard/apigateway/apigateway/apis/web/resource/serializers.py [hunk 2]
```diff
@@ -541,20 +544,23 @@ class SelectedResourceSLZ(serializers.Serializer):
...
+        allow_blank=True,
+        required=False,
+        help_text="文档语言，en: 英文，zh: 中文",
     )
 
 
-class ResourceImportCheckInputSLZ(ResourceImportInputSLZ):
+class ResourceImportDocPreviewInputSLZ(serializers.Serializer):
+    gateway = serializers.HiddenField(default=CurrentGatewayDefault())
+    review_resource = ResourceDataImportSLZ(required=True, help_text="要预览z")
```
[the remark this answers is not in the document]
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/apis/web/resource/serializers.py:563]: done

File: src/dashboard/apigateway/apigateway/apis/web/resource/serializers.py [hunk 3]
```diff
@@ -563,19 +569,44 @@ class ResourceImportCheckInputSLZ(ResourceImportInputSLZ):
...
-    doc = serializers.SerializerMethodField(help_text="资源文档")
-    schema = serializers.SerializerMethodField(help_text="参数协议")
+    match_subpath = serializers.BooleanField(help_text="是否匹配所有子路径")
+    is_public = serializers.BooleanField(help_text="是否公开")
+    allow_apply_permission = serializers.BooleanField(help_text="是否允许应用在开发者中心申请访问资源的权限")
+
+    doc = serializers.SerializerMethodField(help_text="资源文档列表，zh和en")
+    auth_config = ResourceAuthConfigSLZ(help_text="认证配置")
+    backend_name = serializers.SerializerMethodField(help_text="后端服务名称")
+    backend_config = BackendConfigSLZ(help_text="后端配置")
```
[the remark this answers is not in the document]
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/apis/web/resource/serializers.py]: done

**Pull request merged**